tests/e2e-leg-4/pvc-expansion/verify-pvc-change/volume-expansion-disabled/assert.yaml

```diff
@@ -11,12 +11,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: v1
-kind: Event
-reason: SkipPVCExpansion
-source:
-  component: verticadb-operator
-involvedObject:
-  apiVersion: vertica.com/v1
-  kind: VerticaDB
-  name: v-pvc-expansion
+# No event to check if expansion is skipped
```

Comment:
Why do you still keep this file?

Reply:
If we run with a PVC that allows for expansion, then the step can still be used to verify that.